metrics_utility/test/ccspv_reports/test_consistent_indirectly_managed_nodes_output.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
```

Comment:
for testing this, you could just copy the test in https://github.com/ansible/metrics-utility/blob/devel/metrics_utility/test/ccspv_reports/test_complex_CCSPv2_with_indirect.py

add the extra sheet breaking it and making sure it passes the same asserts (so you can e.g. extract the asserted values into 1 place, to ensure the content is the same)